core/unix/coredump.c

```diff
@@ -274,6 +273,23 @@ mcontext_to_user_regs(DR_PARAM_IN priv_mcontext_t *mcontext,
 #endif
 }
 
+/*
+ * Return the entry point of the program.
+ */
+uint64_t
+get_entry_point(DR_PARAM_IN priv_mcontext_t *mcontext)
+{
+#ifdef DR_HOST_NOT_TARGET
+    return 0;
```

Comment:
Is this mode supported?  Should there be an error on trying to create a core dump in this mode?